lib/gui/modules/analytics.js

```diff
@@ -129,11 +129,8 @@ analytics.service('AnalyticsService', function($log, $window, $mixpanel, Setting
 
     }
 
-    if (data) {
-      message += ` (${JSON.stringify(data)})`;
-    }
-
-    this.logDebug(message);
+    const debugMessage = data ? `${message} (${JSON.stringify(data)})` : message;
```

Comment:
IMHO the old version was more readable, but I guess this is the price you pay for banning string concatenation ;)